electron_app/src/components/History.vue

```diff
@@ -3,7 +3,7 @@
     <div  class="animatable_content_box ">
     
         <div v-if="Object.values(app_state.history).length > 0">
-            <div v-for="history_box in Object.values(app_state.history).reverse()" :key="history_box.key" style="clear: both;">
+            <div v-for="history_box in Object.values(app_state.history)" :key="history_box.key" style="clear: both;">
```

Comment:
why remove the .reverse ?

Reply:
I must have made a mistake while rebasing

Reply:
BTW why history is stored as an object with random keys instead of array?